development/generate-lavamoat-policies.js

```diff
@@ -0,0 +1,53 @@
+const concurrently = require('concurrently');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { BuildType } = require('./lib/build-type');
+
+start().catch((error) => {
+  console.error('Policy generation failed.', error);
+  process.exitCode = 1;
```

Comment:
We can exit immediately? 
```suggestion
  process.exit(1);
```

Reply:
We can, but we probably shouldn't unless necessary, see: https://nodejs.org/api/process.html#process_process_exit_code